apb-base/broker-bind-creds

```diff
@@ -2,9 +2,14 @@
 
 CREDS="/etc/apb/bind-creds"
 
+# Broker will pickup exit codes during retries
+#
+# Exit code 0 - Credentials found
+# Exit code 2 - Credentials not available
+#
 if [ ! -s $CREDS ]; then
     echo "Bind credentials not available."
-    exit 1
+    exit 2
```

Comment:
+1 I'm ok with this. 0 is good. 1 isn't a good idea because any command returns 1 on error. So 2 is a good choice to be different and non-zero.